nucliadb_utils/nucliadb_utils/settings.py

```diff
@@ -207,3 +207,7 @@ class NATSConsumerSettings(BaseSettings):
 
 
 nats_consumer_settings = NATSConsumerSettings()
+
+
+def is_onprem_nucliadb() -> bool:
```

Comment:
Should we use `nucliadb_utils.settings.nuclia_settings.onprem` instead?

Reply:
I think this one is safer, as we can't control which env vars the user will set.
For sure if a nua_key is configured, it means we are in onprem.